src/Surfnet/StepupRa/RaBundle/Resources/views/Vetting/Sms/provePossession.html.twig

```diff
@@ -20,10 +20,35 @@
 
     <hr>
 
-    {{ form(form) }}
+    {{ form_start(form) }}
+    <div class="row">
+        {{ form_errors(form) }}
+        <div class="col-md-4">
+            {{ form_label(form.challenge, null, {'label_attr': {'class': 'pull-right'}}) }}
+        </div>
+        <div class="col-md-8">
+            {{ form_widget(form.challenge) }}
+        </div>
+    </div>
+    <div class="row">&nbsp;</div>
+    <div class="row offset-top">
+        <div class="col-md-1">
+            &nbsp;
+        </div>
+        <div class="col-md-5">
+            {{ form_row(form.resendChallenge) }}
+        </div>
+        <div class="col-md-3">
+            <button onclick="redirectWithPostTo({{ path('ra_vetting_cancel', { procedureId: app.request.attributes.get('procedureId') })|json_encode }}); return false" class="btn btn-danger pull-right" formnovalidate="formnovalidate">
```

Comment:
Fair point, I thought of a better alternative that I just implemented.
You should like this better. Could you rereview?